vcd/writer.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, file, timescale='1 us', date=None, comment='',
         self._scope_var_names = {}
         self._scope_types = {}
         self._ident_values = OrderedDict()
+        self._init_timestamp = init_timestamp
         self._prev_timestamp = 0
```

Comment:
I had tried this. The problem I hit is that the signal creation/initialization technically occurs at time 0 and there are some checks that `0 < self.prev_timestamp` which will fail if we initialze `self.prev_timestamp` to something other than 0.